docs/source/guides/auth.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: Authorization with Apollo MCP Server
+---
+
+<PlanRequired plans={["Developer", "Standard", "Enterprise"]}/>
+
+The Apollo MCP server supports authorizing clients (e.g. LLMs) in accordance with [the MCP specification](https://modelcontextprotocol.io/specification/2025-06-18/basic/authorization).
+
+To implement authorization, you need:
+- An [OAuth 2.1-compliant](https://oauth.net/2.1/) Identity Provider (for example, your own in-house IdP or a third-party IdP such as Auth0, Okta, Keycloak, etc)
+- An Apollo Router [configured to use JWT authentication](/graphos/routing/security/jwt)
+- A configuration file for the MCP server that [defines the `auth` settings](/apollo-mcp-server/command-reference#auth)
+
+## Example with Auth0 and MCP Inspector
+
+To follow along with this example, you need to:
+- Clone the [Apollo MCP Server repo](https://github.com/apollographql/apollo-mcp-server)
+- Install [Apollo Rover CLI](/rover/getting-started) v0.35 or later
+
+### Set up the Auth0 Identity Provider
+
+#### Create the Auth0 API
+
+1. Create an Auth0 API by navigating to `Applications -> APIs`. Click **Create API** and fill in the fields necessary. You'll need to note the `Identifier` you choose, as it will be used in the MCP server configuration later on as the `audience` property.
+1. Navigate to your dashboard **Settings**.
```

Comment:
I'm afraid users will get lost right from the start here because the Auth0 dashboard can be overwhelming. It might help to spell out the navigation steps, for example: "Navigate to `Dashboard -> Settings -> Advanced`."